Explorer/Assets/DCL/Character/CharacterPreview/CharacterPreviewController.cs

```diff
@@ -81,6 +83,19 @@ public void UpdateAvatar(CharacterPreviewAvatarModel avatarModel)
                 PartitionComponent.TOP_PRIORITY);
 
             avatarShape.IsDirty = true;
+
+            return WaitForAvatarInstantiatedAsync(ct);
```

Comment:
Is it intended for this to not be awaited?

Reply:
yup